src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -143,6 +157,8 @@ export default class SourceFileLinter {
 			ts.forEachChild(node, visitMetadataNodes);
 		} else if (this.isUi5ClassDeclaration(node, "sap/ui/core/Control")) {
 			this.analyzeControlRendererDeclaration(node);
+		} else if (ts.isPropertyAssignment(node) && removeQuotes(node.name.getText()) === "theme") {
```

Comment:
Checking for each individual node type would allow accessing the un-quoted name, but requires more code / checks. I'm fine with this here.

I anyways considered looking into making such checks easier in general by creating a utility function, as we have such a use case in multiple places.